apps/src/templates/SchoolNotFound.jsx

```diff
@@ -156,6 +161,18 @@ export default class SchoolNotFound extends Component {
       <div style={styles.errors}>{i18n.schoolInfoRequired()}</div>
     );
 
+    // Check if country selected is US or non-US.  This new check
+    // is used to decide which fields in the school info interstitial
+    // form should be required.
+    let countryIsNotUS = false;
+    if (this.props.country && this.props.country === 'United States') {
+      countryIsNotUS = true;
+    }
+    let isRequired = countryIsNotUS;
```

Comment:
nit: what's required? Can you add either a comment or rename this variable to be a bit more descriptive?

Reply:
Changed variable to isLabelRequired(which is more descriptive) and add a comment.

Reply:
Thanks!